src/Hazelcast.Net.DocAsCode/GatherHazelcastOptions.cs

```diff
@@ -39,12 +40,13 @@ static bool IsOptionsModel(string modelKey)
                 => modelKey.StartsWith("~/obj/dev/api/") &&
                    modelKey.EndsWith("Options.yml");
 
-            Logger.LogInfo("Gathering options.");
+            Logger.LogInfo(" Gathering option classes.");
```

Comment:
Is whitespace at the message start here and below intended?

Reply:
No, fixing.